canonical-websites/design.ubuntu.com
# Remove text section

## Done

Removed the `/brand/text` page

## QA

 - `./run`
 - See that /brand/text no longer exists
 - See that there are no references to the page on /brand
